src/sentry/integrations/slack/service.py

```diff
@@ -283,20 +270,14 @@ def _handle_parent_notification(
                     text=notification_to_send,
                     blocks=json_blocks,
                 )
-                # TODO(iamrajjoshi): Remove this after we validate lifecycle
-                metrics.incr(SLACK_ACTIVITY_THREAD_SUCCESS_DATADOG_METRIC, sample_rate=1.0)
             except SlackApiError as e:
-                # TODO(iamrajjoshi): Remove this after we validate lifecycle
-                self._logger.info(
-                    "failed to post message to slack",
-                    extra={"error": str(e), "blocks": json_blocks, **extra},
-                )
-                metrics.incr(
-                    SLACK_ACTIVITY_THREAD_FAILURE_DATADOG_METRIC,
-                    sample_rate=1.0,
-                    tags={"ok": e.response.get("ok", False), "status": e.response.status_code},
+                lifecycle.add_extras(
+                    {
+                        "rule_action_uuid": parent_notification.rule_action_uuid,
+                        "channel_id": channel_id,
+                        "thread_ts": parent_notification.message_identifier,
+                    }
                 )
-                lifecycle.add_extras({"rule_action_uuid": parent_notification.rule_action_uuid})
```

Comment:
does this need to be kept? it's part of `lifecycle`

Reply:
whoops ur right